controllers/main.js

```diff
@@ -146,18 +146,28 @@ module.exports.renderDetailedResults = async (req, res, next) => {
 
 module.exports.sendPlayData = async (req, res, next) => {
   const id = uuidv4();
-  const playSession = new PlaySession({
-    userId: req.user?._id || "guest",
-    sessionId: id,
-    difficulty: req.body.difficulty,
-    length: req.body.sessionLength,
-    sessionData: [],
-    sessionEnded: false,
-    sessionStartTime: Date.now(),
-  });
-  await playSession.save();
-  res.redirect(`/play/${id}/1`);
-  next();
+  const difficulty = parseInt(req.body.difficulty);
+  const length = parseInt(req.body.sessionLength);
+  if (
+    !(difficulty in remapDifficultyTexts) ||
+    !lengthSettingsOptions.includes(length)
+  ) {
+    console.log("selected settings are invalid");
```

Comment:
If you intend to keep this console log, then consider adding more information to it. Imagine this scenario:

You are remotely checking your server's terminal after deploying and you see this console log which does not really say anything, it's basically just a string with no information. Maybe console logging the session settings for example would be good as well?